src/RendererConfig.h

```diff
@@ -16,9 +16,9 @@ struct RendererConfig {
     std::string jobId = "0";
     unsigned int samplesPerPixel = 20; 
     unsigned int recursionDepth = 3; 
-    std::string objPath = "models/cornell-box.obj"; 
-    unsigned int gpuNumber = 2;   
-    unsigned int streamsPerGpu = 5;
+    std::string path = "models/Duck/glTF/Duck.gltf";
```

Comment:
We shouldn't set a default value here since `ArgumentLoader` should handle that case.